projects/js-packages/publicize-components/src/hooks/use-publicize-config/index.js

```diff
@@ -99,6 +100,11 @@ export default function usePublicizeConfig() {
 	 */
 	const isAutoConversionEnabled = !! getJetpackData()?.social?.isAutoConversionEnabled;
 
+	/**\
+	 * Returns true if the post type is a Jetpack Social Note.
+	 */
+	const isJetpackSocialNote = 'jetpack-social-note' === currentPostType;
```

Comment:
We could put this directly in the return statement below, but I doubt it makes much difference.

Reply:
Yeah, I put it like this to follow the convention here and to be able to add a doc comment to it.